charts/openfga/values.yaml

```diff
@@ -254,3 +254,4 @@ sidecars: []
 migrate:
   sidecars: []
   annotations: {}
+  labels: {}
```

Comment:
@aaronschweig mind adding a new line at eof here?

Reply:
Done in b8f2378